app/src/main/java/com/duckduckgo/app/browser/BrowserChromeClient.kt

```diff
@@ -169,9 +171,11 @@ class BrowserChromeClient @Inject constructor(
     }
 
     override fun onPermissionRequest(request: PermissionRequest) {
```

Comment:
are you going to add a testcase for this one?

Reply:
yes, tests will be added as soon as they are finished 👍